app/src/main/java/com/training/simpleloginform/MainActivity.java

```diff
@@ -34,8 +47,18 @@ protected void onCreate(Bundle savedInstanceState) {
         submitButton = findViewById(R.id.submit_button);
         resetButton = findViewById(R.id.reset_button);
 
+
+            names.add("Desmond");
```

Comment:
Can we use the Names class? Just copy it from the previous code samples and use it here :)